subnet_router.go

```diff
@@ -0,0 +1,98 @@
+package packngo
+
+import (
+	"path"
+)
+
+const subnetRouterBasePath = "/subnet-routers"
+
+type SubnetRouterService interface {
+	List(projectID string, opts *ListOptions) ([]SubnetRouter, *Response, error)
+	Create(projectID string, input *SubnetRouterCreateRequest) (*SubnetRouter, *Response, error)
+	Get(subnetRouterID string, opts *GetOptions) (*SubnetRouter, *Response, error)
+	Delete(subnetRouterID string) (*Response, error)
+}
+
+type SubnetRouter struct {
+	ID             string                `json:"id"`
+	State          string                `json:"state"`
+	Project        *Project              `json:"project,omitempty"`
+	VirtualNetwork *VirtualNetwork       `json:"virtual_network,omitempty"`
+	IPReservation  *IPAddressReservation `json:"ip_reservation,omitempty"`
+	Href           string                `json:"href"`
+	Created        string                `json:"created_at,omitempty"`
```

Comment:
We've been inconsistent about naming the field after the JSON field. Many cases of calling "created_at" both `Created` and `CreatedAt`.

Perhaps we should match the json field? (likewise for Updated)

Reply:
Sounds good